include/morphio/properties.h

```diff
@@ -106,40 +125,30 @@ struct MitochondriaPointLevel
 
     MitochondriaPointLevel() {}
     MitochondriaPointLevel(const MitochondriaPointLevel& data,
-        SectionRange range);
+                           SectionRange range);
 
     MitochondriaPointLevel(
         std::vector<uint32_t> sectionId,
         // relative pathlength between the current points
         // and the start of the neuronal section
         std::vector<MitoPathLength::Type> relativePathLengths,
         std::vector<MitoDiameter::Type> diameters);
+
+    bool diff(const MitochondriaPointLevel& other, LogLevel verbose) const;
```

Comment:
Maybe it needs to be renamed from `verbose` to `logLevel` now.
Also, as discussed verbally, another option is an environment variable controlling log verbosity for all `MorphIO` methods.